tests/test_datastore.py

```diff
@@ -7,11 +7,11 @@
 class TestDatastore(TestBase):
 
     def setUp(self):
-        pickle_patcher = patch('virtwho.datastore.pickle')
-        self.mock_pickle = pickle_patcher.start()
-        self.mock_pickle.dumps.return_value = sentinel.pickled_value
-        self.mock_pickle.loads.return_value = sentinel.unpickled_value
-        self.addCleanup(pickle_patcher.stop)
+        copy_patcher = patch('virtwho.datastore.copy')
+        self.mock_copy = copy_patcher.start()
+        self.mock_copy.deepcopy.side_effect = [sentinel.deep_copy_value_1,
+                                                sentinel.deep_copy_value_2]
```

Comment:
Wrong indentation.

BTW: confusing that you can use: `sentinel.any_strange_name_of_attribute` without raising exception.